### ISSUE TITLE
[LinkQuoter] Add an option to toggle the `delete_message` button.

### ISSUE THREAD

**Issue by sgoudham** [AAA3A-AAA3A/AAA3A-cogs#48]
### What cog is this feature request for?

linkquoter

#### Proposal

I would like to be able to toggle the `delete_message` button appearing on link quoted messages, so that they don't appear in the first place.

Maybe an extra option like `showdeletemessagebutton` or equivalent? This could be set to `True` as the default, mimicking the current behaviour.

Relevant code block that would need to be "toggle-able": https://github.com/AAA3A-AAA3A/AAA3A-cogs/blob/a398409ea8a148081ae01365aa9625d1c7955fc4/linkquoter/linkquoter.py#L33-L36

**Comment by sgoudham** [comment]
Thank you so much! <3 @AAA3A-AAA3A

**Comment by sgoudham** [comment]
Hi again... I've just ran the following commands

```
cog update linkquoter
reload linkquoter
setlinkquoter deletemessagebutton False
```

and unfortunately receiving the following error message when quoting a message:

```
Error in [hybrid|text] command 'linkquote'. Check your console or logs for details. If necessary, please inform the creator of the cog in which this command is located. Thank you.
You can send this error to the developer by running the following command:
~AAA3A_utils senderrorwithsentry b82ac24feb29464290b0e0f7d645cd45
```

Would you happen to know what's going wrong here?

**Comment by AAA3A-AAA3A** [comment]
No need to recreate a bug report. lol Should be fixed with `3a8ee9b6`.

**Comment by backwardspy** [comment]
works perfectly, thank you!
